README.md

```diff
@@ -14,3 +14,8 @@
 [docs-link]: https://docs.rs/proxy-wasm
 [license-badge]: https://img.shields.io/github/license/proxy-wasm/proxy-wasm-rust-sdk
 [license-link]: https://github.com/proxy-wasm/proxy-wasm-rust-sdk/blob/master/LICENSE
+
+## Examples that use this SDK
```

Comment:
Could you create `Examples` section that points to the existing files in `examples/` directory, and rename this section to `Articles & blog posts from the community`? It's fine if you want to do only the latter part.

Reply:
Sure! I'll do this today.

Reply:
Done:

+ Renamed to create "Articles & blog posts from the community" (866776051e7f7a43624d0c776affc1308332277e)
+ Added "Examples" referencing same (9a7d5e71449bab436ce35501f9042497ef6fda0d)